src/containers/TabularResourceViewContainer.js

```diff
@@ -14,10 +16,11 @@ export class TabularResourceViewContainer extends React.Component {
     }
   }
 
+  //Takes a single resource and returns Handsontable spec
   buildHandsontableSpec(data) {
     let spec = {
-      data: data.slice(1),
-      colHeaders: data[0],
+      data: data.slice(1), //excluding headers
```

Comment:
Minor stuff for the future: it is better to divide trailing comments from the code by 2 spaces at least. This is from pep8, but probably applies to js too.